src/NServiceBus.AcceptanceTests/Recoverability/Retries/When_doing_flr_with_native_transactions.cs

```diff
@@ -14,9 +15,16 @@ public void Should_do_5_retries_by_default_with_native_transactions()
             Scenario.Define(() => new Context { Id = Guid.NewGuid() })
                     .WithEndpoint<RetryEndpoint>(b => b.Given((bus, context) => bus.SendLocal(new MessageToBeRetried { Id = context.Id })))
                     .AllowExceptions()
-                    .Done(c => c.ForwardedToErrorQueue || c.NumberOfTimesInvoked > 5)
+                    .Done(c => c.ForwardedToErrorQueue)
                     .Repeat(r => r.For(Transports.Default))
-                    .Should(c => Assert.AreEqual(5+1, c.NumberOfTimesInvoked, "The FLR should by default retry 5 times"))
+                    .Should(c =>
```

Comment:
I think we decided at some point to try to avoid `Should` and `Repeat` and try to write tests in such a way so that we can just add asserts to the test method. Do you remember why @danielmarbach @andreasohlund ?